editor/src/clj/editor/properties_view.clj

```diff
@@ -215,17 +231,21 @@
                 comp (doto (create-grid-pane children)
                        (GridPane/setConstraints index 0)
                        (GridPane/setHgrow Priority/ALWAYS))
+                cancel-fn (fn [_]
+                            (let [property (property-fn)
+                                  current-vals (properties/values property)]
+                              (update-ui-fn current-vals
+                                            (properties/validation-message property)
+                                            (properties/read-only? property))))
                 update-fn (fn update-fn [_]
                             (let [property (property-fn)
                                   current-vals (properties/values property)
                                   old-num (ffirst current-vals)
                                   num (parse-num (.getText text-field) old-num)]
-                              (if num
+                              (if (and num (not= num old-num))
```

Comment:
this small check fixes issue with not adding `0` in  the end (the same for Num properties)

https://github.com/defold/defold/assets/2209596/a3fe35b1-62f7-4869-8a94-56e0718bc7cd